src/Client.hs

```diff
@@ -0,0 +1,28 @@
+module Client (client) where
+
+import           Prelude ()
+import           Prelude.Compat
+import           Network.HTTP.Client
+import           Network.HTTP.Client.Internal
+import           Network.Socket hiding (recv)
+import           Network.Socket.ByteString (sendAll, recv)
+import qualified Data.ByteString.Lazy as L
+
+import           Http (newSocket, socketAddr)
+
+client :: IO L.ByteString
+client = simpleHttp "http://typeful.net"
```

Comment:
Haha, in fact it does not really matter what we put here, but I'll change it to localhost.